data/json/items/crossbows.json

```diff
@@ -44,23 +44,67 @@
       "type" : "AMMO",
       "id" : "bolt_steel",
       "price" : 5000, "//" : "That's the going rate for aluminum, but unless you want carbon fiber, that's as metal as Cabela's seems to stock.",
-      "name" : "steel crossbow bolt",
+      "name": "steel broadhead bolt",
       "symbol" : "=",
       "color" : "green",
       "description" : "A sharp bolt made from steel.  Deadly in skilled hands.  Stands an excellent chance of remaining intact once fired.",
       "material" : "steel",
       "volume" : 1,
-      "weight" : 217,
+      "weight" : 108,
       "bashing" : 5,
       "ammo_type" : "bolt",
-      "damage" : 20,
+      "damage" : 12,
       "pierce" : 10,
-      "range" : 6,
-      "dispersion" : 120,
+      "range" : 18,
+      "dispersion" : 90,
       "loudness": 0,
       "count" : 10,
-      "effects" : ["RECOVER_25"]
+      "effects" : ["RECOVER_50"]
    },
+   {
+	  "type": "AMMO",
+	  "id": "bolt_steel_bodkin",
+	  "price": 4000,
+	  "name": "steel bodkin bolt",
+	  "symbol": "=",
+	  "color": "green",
+	  "description": "A sharp bolt made from steel with an armor piercing bodkin tip.  Deadly in skilled hands. Stands an excellent chance of remaining intact once fired.",
+	  "material": "steel",
+	  "volume": 1,
+	  "weight": 95,
+	  "bashing": 3,
+	  "cutting": 2,
+	  "ammo_type": "bolt",
+	  "damage": 8,
+	  "pierce": 11,
+	  "range": 16,
+	  "dispersion": 85,
+	  "loudness": 0,
+	  "count": 10,
+	  "effects": ["RECOVER_50"]
+   },
+   {
+	  "type": "AMMO",
+	  "id": "bolt_steel_target",
+	  "price": 4000,
+	  "name": "steel target bolt",
+	  "symbol": "=",
+	  "color": "green",
+	  "description": "A sharp bolt made from steel with extra long fletching and a round, balanced tip for extra accuracy, at the cost of some damage. Deadly in skilled hands. Stands an excellent chance of remaining intact once fired.",
+	  "material": "steel",
+	  "volume": 1,
+	  "weight": 122,
+	  "bashing": 3,
+	  "cutting": 2,
+	  "ammo_type": "bolt",
+	  "damage": 10,
+	  "pierce": 7,
+	  "range": 12,
```

Comment:
Again, is the target variants supposed to have higher or lower ranges than others?

>Adds bodkin (armor piercing) and target (accuracy/range bonus) variants

Reply:
Fixed.